apps/examples/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -64,10 +64,18 @@
 #include <crc16.h>
 #include <sys/ioctl.h>
 #include <sys/stat.h>
+#include <debug.h>
 
 #include <tinyara/fs/ioctl.h>
 #include <tinyara/config.h>
 
+/* Debug statement definition for smartfs powercut test app */
+#ifdef CONFIG_DEBUG_FS_ERROR
```

Comment:
It is application, so this is not good choice.
I think we'd better define debug level here.
please see below.
```
#define DEBUG_NONE 0
#define DEBUG_LOW   1
#define DEBUG_HIGH  2

#define DEBUG_LEVEL DEBUG_LOW //we can change this in code level.
#if (DEBUG_LEVEL == DEBUG_LOW)
#define PCDBG(format, ...)   printf(EXTRA_FMT format EXTRA_ARG, ##__VA_ARGS__)
#define PCVDBG(format, ...)
#elif (DEBUG_LEVEL == DEBUG_HIGH)
#define PCDBG(format, ...)   printf(EXTRA_FMT format EXTRA_ARG, ##__VA_ARGS__)
#define PCVDBG(format, ...) printf(EXTRA_FMT format EXTRA_ARG, ##__VA_ARGS__)
#else // NONE
#define PCDBG(format, ...)
#define PCVDBG(format, ...)
#endif
```